src/Uno.UWP/System/Profile/AnalyticsInfo.cs

```diff
@@ -0,0 +1,20 @@
+﻿namespace Windows.System.Profile
```

Comment:
🎗️ Reminder: Create iOS and Android documentation that explains the implementation.